src/hooks/useHomeShortcuts.js

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from 'react'
+import { Q, useClient } from 'cozy-client'
+import { useI18n } from 'cozy-ui/react/I18n'
+import get from 'lodash/get'
+
+const useHomeShortcuts = () => {
+  const client = useClient()
+  const { t } = useI18n()
+  const [files, setFiles] = useState([])
+
+  useEffect(() => {
+    const load = async () => {
+      const folder = await client.query(
+        Q('io.cozy.files').where({ path: t('magic_folder_path') })
```

Comment:
I don't think `magic_folder_path` speaks for itself. magic_folder_for_config_path or something? We already have a few other "magic folder"